src/charm.py

```diff
@@ -12,15 +12,13 @@
 
 import logging
 from pathlib import Path
-import os
-from requests.exceptions import ConnectionError
+from requests.exceptions import ConnectionError as RequestsConnectionError
 from urllib3.exceptions import NewConnectionError, MaxRetryError
 import time
 import re
 
-from ops.charm import CharmBase
-from ops.main import main
-from ops.framework import StoredState
+import ops
+from ops.charm import CharmBase, ActionEvent
```

Comment:
Isn't this importing same stuff twice?

Reply:
Well kind of, I'll clean it up before merging